src/main/java/me/zeroeightsix/kami/module/modules/render/BreakingESP.kt

```diff
@@ -25,9 +27,14 @@ import net.minecraft.util.math.BlockPos
         category = Module.Category.RENDER
 )
 class BreakingESP : Module() {
-    private val ignoreSelf = register(Settings.b("IgnoreSelf", false))
+    private val espSelf = register(Settings.b("ESPSelf", true))
+    private val warnSelf = register(Settings.b("WarnSelf", false))
     private val obsidianOnly = register(Settings.b("ObsidianOnly", false))
     private val warning = register(Settings.b("Warning", false))
+    private val warningProgress = register(Settings.integerBuilder("Warning Progress").withMinimum(0).withValue(4).withMaximum(9).build())
+    private val chatWarn = register(Settings.b("Chat Warning", true))
```

Comment:
should be off by default imo

Reply:
makes sense

Reply:
Which warning type should be on by default? Currently chat warning is the only one that defaults to on

Reply:
The visual warning